### PR TITLE
tech: static factory method for creating DatabricksSqlResponse objects

### PR DESCRIPTION
**Pull request by JesperGomes**
## References

* #1023

### CODE REVIEW

File: source/dotnet/wholesale-api/CalculationResults/CalculationResults.Infrastructure/CalculationResultClient/CalculationResultClient.cs [hunk 1]
```diff
@@ -92,10 +92,10 @@ private async Task<Table> ExecuteSqlStatementAsync(string sqlStatement)
 
             var databricksSqlResponse = _databricksSqlResponseParser.Parse(jsonResponse);
 
-            if (databricksSqlResponse.State == "SUCCEEDED")
+            if (databricksSqlResponse.State == DatabricksSqlResponseState.Succeeded)
```
Comment (BjarkeMeier) [inline source/dotnet/wholesale-api/CalculationResults/CalculationResults.Infrastructure/CalculationResultClient/CalculationResultClient.cs:95]: I like

File: source/dotnet/wholesale-api/CalculationResults/CalculationResults.Infrastructure/CalculationResultClient/DatabricksSqlResponseFactory.cs [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace Energinet.DataHub.Wholesale.CalculationResults.Infrastructure.CalculationResultClient;
+
+public static class DatabricksSqlResponseFactory
```
Comment (BjarkeMeier) [inline source/dotnet/wholesale-api/CalculationResults/CalculationResults.Infrastructure/CalculationResultClient/DatabricksSqlResponseFactory.cs:17]: This is ok with me. Personally, I think I'd just have added these static methods to the `DatabricksSqlResponse` class itself. Then I can make the ctor private too and thus ensure that nobody uses them directly. And I find it simpler as there is no particular reason in this case and this module to add a separate factory.

Comment (JesperGomes) [reply source/dotnet/wholesale-api/CalculationResults/CalculationResults.Infrastructure/CalculationResultClient/DatabricksSqlResponseFactory.cs]: done

**Review by BjarkeMeier — APPROVED** [review]
Nice

**Pull request merged**